docs/proposals/proposal-audit-log.md

```diff
@@ -0,0 +1,209 @@
+# Audit log
+
+### Purpose of audit log
+
+The audit log is a feature that allows users to track changes made to their organization, projects,
+schemas, and other resources.
+
+## Events to track
+
+### OrganizationManager
+
+- `leaveOrganization`
+- `createOrganization`
+- `deleteOrganization`
```

Comment:
about these, since the audit log lives on the organization level, I am unsure whether we can have these.

Reply:
why not? what if I self host and someone deletes my organisation by mistake? 
I would love to see which user did that.

Reply:
I was talking from a users point of perspective. As admins will look at this and if the organization no longer exist, they would have to contact us directly. Which is fine in that scenario. So let's keep it.